toolkit/scripts/toolchain/build_official_toolchain_rpms.sh

```diff
@@ -357,10 +357,10 @@ chroot_and_install_rpms libxml2
 echo Download JDK rpms
 case $(uname -m) in
     x86_64)
-        wget -nv --timeout=30 https://packages.microsoft.com/cbl-mariner/2.0/prod/Microsoft/x86_64/msopenjdk-11-11.0.14.1+1-LTS-31207.x86_64.rpm --directory-prefix=$CHROOT_RPMS_DIR_ARCH
+        wget -nv --no-clobber --timeout=30 https://packages.microsoft.com/cbl-mariner/2.0/prod/Microsoft/x86_64/msopenjdk-11-11.0.14.1+1-LTS-31207.x86_64.rpm --directory-prefix=$CHROOT_RPMS_DIR_ARCH
```

Comment:
I'm wondering if we really want to just replace...in the even that local copy is bad, this will overwrite....but ok with the fix.  Definitely an improvement.

Reply:
In both cases for this scenario (delta toolchain build) the file is downloaded from PMC, so there should not be an actual local copy. Adding --no-clobber here is a slight perf improvement since the operation is a no-op instead of downloading the relatively large file again.